src/Umbraco.Cms.Api.Management/Controllers/Server/InformationServerController.cs

```diff
@@ -11,21 +9,14 @@ namespace Umbraco.Cms.Api.Management.Controllers.Server;
 public class InformationServerController : ServerControllerBase
 {
     private readonly ISystemInformationService _systemInformationService;
-    private readonly IUmbracoMapper _mapper;
 
-    public InformationServerController(ISystemInformationService systemInformationService, IUmbracoMapper mapper)
+    public InformationServerController(ISystemInformationService systemInformationService)
     {
         _systemInformationService = systemInformationService;
-        _mapper = mapper;
     }
 
     [HttpGet("information")]
     [MapToApiVersion("1.0")]
-    [ProducesResponseType(typeof(ServerInformationResponseModel), StatusCodes.Status200OK)]
-    public Task<IActionResult> GetServerInformation()
-    {
-        ServerInformationResponseModel responseModel = _mapper.Map<ServerInformationResponseModel>(_systemInformationService.GetSystemInformation())!;
-
-        return Task.FromResult<IActionResult>(Ok(responseModel));
-    }
+    [ProducesResponseType(typeof(Dictionary<string, object>), StatusCodes.Status200OK)]
```

Comment:
Seems wierd to not use a viewmodel for this, not entirely sure if this is the best approach, or if we should switch to a proper ViewModel

Reply:
When we return a ServerInformationResponseModel it should be that model